duoapi.go

```diff
@@ -160,12 +161,16 @@ func NewDuoApi(ikey string,
 	if opts.transport != nil {
 		opts.transport(tr)
 	}
+	newUserAgent := "duo_api_golang/" + version
+	if userAgent != "" {
```

Comment:
Maybe this is just an opinion, but I feel that if the caller explicitly set the user agent, that we shouldn't modify it. Modifying the user agent I pass in would be, to me, unexpected behavior.